media/css/base/page-not-found.scss

```diff
@@ -70,15 +70,17 @@ main {
     .logo-mozilla {
         display: inline-block;
         height: 50px;
-        transform: translateY(-6px);
+        width: 141px;
```

Comment:
I think a better way to do this is to add a `width="140"` attribute to the logo image in `bedrock/base/templates/base/base-error.html` rather than setting the width in CSS.